### ISSUE TITLE
Text invisible while editing when centered or right-aligned

### ISSUE THREAD

**Issue by jancborchardt** [penpot/penpot#887]
**Describe the bug**
While editing text, if it is centered or right-aligned, it will be invisible.

**To Reproduce**
1. Open a text field
2. Put in some text
3. Change it from left-aligned to right-aligned
4. See it vanish

**Expected behavior**
The text should not vanish 👻

**Screenshots**
Here’s a short gif:
![Text vanishes depending on alignment](https://user-images.githubusercontent.com/925062/116711900-9650bf80-a9d3-11eb-830e-7481d0118661.gif)

**Desktop (please complete the following information):**

-   OS: Ubuntu 20.04 LTS
-   Browser: Happens in both latest Firefox and Chromium

**Environment (please complete the following information):**
Using SAAS (https://design.penpot.app)

**Comment by niwinz** [comment]
Hi @jancborchardt

I am unable to reproduce this behavior.

This can be a browser issue or penpot issue, for discarding  the second, can you create a isolated file in your drafts or somewhere and pass here the workspace url. So we can locate the file in the database and check if something is wrong. Leave the file with the text when is invisible.

Thanks

**Comment by fyrak1s** [comment]
Hello, I have the same issue here. I can reproduce it only in chromium (brave browser more specifically, adblocking disabled).
One I go back to my projects and reopen the drafts file the issues is gone. However it is really a bad experience for deigning text.

I can reproduce it when I enter a multiline text and then reallign it to the left the middle or the right. Often some character end up missing.

EDIT: This seems to be a general issue. Now I have entered the text field and changed the color of the first line. As soon as I go out of edit mode, the line "disappears".

**Comment by fyrak1s** [comment]
@niwinz how can I share a test case with you?

EDIT: is this the workspace url? https://design.penpot.app/#/workspace/ccba2140-7bda-11ec-9ea7-96385a0e391f/90d48b10-91b1-11ec-bd38-efdb6fa63305?page-id=26690b51-7bdd-11ec-9ea7-96385a0e391f

Here is what it looks like this when I make edits to the content:
![penpot_testcase](https://user-images.githubusercontent.com/2538022/154814262-d4d081d4-ea7c-40df-abfa-5495a20c726a.png)
It is easiest to reproduce problems, when you change the color of the first line.

UPDATE:
I have nailed it down to the experimental flag that I had enabled in my browser:
`#enable-raw-draw`

When I disable that flag, everything works as it should :)
So on my side, problem solved.
